Services/GEV/DecentralTrainings/classes/class.gevDecentralTrainingGUI.php

```diff
@@ -1634,7 +1634,13 @@ protected function getFormValuesByTemplateId($a_template_id, $trainer_ids) {
 				foreach ($sup_ids as $sup_id) {
 					$pers_org_unit = $uvg_org_units->getOrgUnitIdOf($sup_id);
 					if($pers_org_unit) {
-						$above_ref_id = gevOrgUnitUtils::getBDOf(gevObjectUtils::getRefId($pers_org_unit));
+
```

Comment:
Das gehört hier eigentlich nicht rein, oder? Ich glaube auch, dass das so nicht richtig ist. Auf die Art wird eine null in $trainer_orgus eingefüllt und verursacht woanders havoc. Besser wäre, erst garnichts an $trainer_orgus anzuhängen, wenn es keine BD gibt.